app/src/test/java/edu/ucandroid/weathernotice/CountryUnitTest.kt

```diff
@@ -17,7 +17,7 @@ import java.net.URL
  *
  * See [testing documentation](http://d.android.com/tools/testing).
  */
-class ExampleUnitTest {
+class CountryUnitTest {
```

Comment:
Refactored to better reflect what part of the code the class is testing

Reply:
Thank you